superset/config.py

```diff
@@ -1100,7 +1104,3 @@ class CeleryConfig:  # pylint: disable=too-few-public-methods
     except Exception:
         logger.exception("Found but failed to import local superset_config")
         raise
-
-# It's possible to add a dataset health check logic which is specific to your system.
```

Comment:
config list below won't be able to be overwritten. Have to move it up.

Reply:
Maybe add a comment block here to stop others from adding new config below this line?

```
# -------------------------------------------------------------------
# *                WARNING:  STOP EDITING  HERE                    *
# -------------------------------------------------------------------
# Don't add config values below this line since local configs won't be
# able to override them.
```

Reply:
+1 for that warning, we've seen issues with config values being added after overrides are loaded.

Reply:
fixed!